src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -406,14 +406,13 @@ public bool WriteIndented
         /// <summary>
         /// Defines how references are treated when reading and writing JSON, this is convenient to deal with circularity.
```

Comment:
Should we get rid of `this is convenient to deal with circularity.`? I think is quite useful if someone steps-into the property on intellisense or catches it reading the docs while trying to find how to avoid an infinite loop.

Reply:
Leaving it is fine. Up to you.